deploy.sh

```diff
@@ -79,6 +80,7 @@ elif [[ "$1" == staging ]]; then
   DATASTORE_NAMESPACE=scraper
   CLUSTER=scraper-cluster
   ZONE=us-central1-a
+  EXTERNAL_IP=35.185.76.159
```

Comment:
Looking in the GCP console, I don't think this IP is the one we want. Can you double check?

Reply:
Well caught!